po/clubhouse.pot

```diff
@@ -0,0 +1,175 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the clubhouse package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: clubhouse\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-11-05 13:25+0100\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: data/achievement-summary-view.ui:33
+msgid "Achievement Badge Name"
+msgstr ""
+
+#: data/achievement-summary-view.ui:49
+msgid "Description"
+msgstr ""
+
+#: data/achievements-view.ui:66
+msgid "Achievements"
+msgstr ""
+
+#: data/achievements-view.ui:131
+msgid "page0"
+msgstr ""
+
+#: data/achievements-view.ui:163
+msgid "page1"
+msgstr ""
```

Comment:
These has no meaning. Maybe "First page", "Second page" ? Or "List of achievements" / "Achievement detail"?